src/components/roverSelector/RoverSelectorPage.tsx

```diff
@@ -1,9 +1,24 @@
 import React from "react";
+import "./RoverSelectorPage.scss";
+import Opportunity from "/images/OpportunityRover.jpg";
+import Spirit from "/images/SpiritRover.jpg";
+import Curiosity from "/images/CuriosityRover.jpg";
 
 export const RoverSelectorPage: React.FunctionComponent = () => {
   return (
-    <main>
-      hello
-    </main>
+    <div className="rover-selector">
+      <div className="rover curiosity">
```

Comment:
Do the names of the rovers need to be classes? In other words, does the CSS care _which_ rover is being displayed? If not:
```suggestion
      <div className="rover">
```
for all 3 please!